themes/root/templates/Helpers/icons/unicode.phtml

```diff
@@ -0,0 +1 @@
+<span class="icon icon--font icon--unicode" role="img" aria-hidden="true">&#x<?=$this->transEsc($this->icon)?>;</span>
```

Comment:
I was thinking `escapeHtml` rather than `transEsc` -- was there a reason you chose the translated version?

Reply:
Also, all the other icon templates offer a mechanism for passing through additional classes and attributes. Did you intentionally avoid aligning with the other icon templates in those regards?

Reply:
Oops and oops. Somewhere along the road I realized we don't need the `::before` trick to display the icon, and then I lost all attribute support. Should be fixed now.